frontend/src/components/custom-tools/prompt-card/Header.jsx

```diff
@@ -224,7 +223,7 @@ function Header({
           <PromptRunBtnSps
             spsLoading={spsLoading}
             handleSpsLoading={handleSpsLoading}
-            handleGetOutput={handleGetOutput}
+            handleGetOutput={() => {}}
```

Comment:
@tahierhussain Do we need this void func here?

Reply:
@jagadeeswaran-zipstack Yes. This needs to be addressed for the SPS. I have kept the SPS component changes on hold and will address them in a separate PR, as it is not a blocker.